src/protocols/vnc/settings.h

```diff
@@ -300,6 +300,11 @@ typedef struct guac_vnc_settings {
      */
     int wol_wait_time;
 
+    /**
+    * Whether or not to write the disable remote input message to the remote host
+    */
```

Comment:
I should also point out that something happened here to misalign the block comment.

Reply:
Should hopefully be fixed.